templates/master.yaml.tmpl

```diff
@@ -2166,6 +2178,17 @@ storage:
                 done
             done
 
+            # apply priority classes
+            PRIORITY_CLASSES_FILE="priority_classes.yaml"
+
+            while
+                $KUBECTL apply -f /srv/${PRIORITY_CLASSES_FILE}
+                [ "$?" -ne "0" ]
+            do
+                echo "failed to apply /srv/${PRIORITY_CLASSES_FILE}, retrying in 5 sec"
```

Comment:
Just to understand here. Why here would `kubectl` fail?

Reply:
Dunno. Flapping api, flapping network, etc